packages/frontend-shared/cypress/e2e/support/e2eSupport.ts

```diff
@@ -266,6 +266,9 @@ function startAppServer (mode: 'component' | 'e2e' = 'e2e') {
 
         await isInitialized.promise
 
+        if (!ctx.lifecycleManager.browsers?.[0]) throw new Error('No browsers available in startAppServer')
```

Comment:
This is a more usual way of doing this check. 

```suggestion
        if (!ctx.lifecycleManager.browsers?.length) throw new Error('No browsers available in startAppServer')
```